source/distributed/grid_refinement.inst.in

```diff
@@ -59,8 +60,9 @@ namespace parallel
       refine_and_coarsen_fixed_number<deal_II_dimension-1,dealii::Vector<S>,deal_II_dimension>
       (parallel::distributed::Triangulation<deal_II_dimension-1,deal_II_dimension> &,
        const dealii::Vector<S> &,
-       const double,
-       const double);
+       double,
+       double,
```

Comment:
But you're just instantiating the function. From the perspective of a function declaration or instantiation, it makes no difference whether a non-reference argument is const or not -- but it makes a difference to the reader.